### ISSUE TITLE
Dashboard is not rendered if trials are pruned without intermediate values

### ISSUE THREAD

**Issue by toshihikoyanase** [optuna/optuna-dashboard#68]
# Bug reports

This might be design, but I found that the dashboard page was not rendered if a study contained pruned trials without intermediate values.

## Expected Behavior

Studies with pruned trials having no intermediate values should be rendered. Or, an error page should be provided if they are not acceptable.

## Current Behavior and Steps to Reproduce

The following is an example code to reproduce the issue. In this example, I pruned trials whose objective values exceed a threshold. A list of studies were rendered as expected, but It showed blank page if I clicked the study name.

![image](https://user-images.githubusercontent.com/3255979/113801406-bfdf4800-9793-11eb-8cc9-b69bdc28119f.png)

```python
import optuna


def objective(trial):
    # Binh and Korn function with constraints.
    x = trial.suggest_float("x", -15, 30)
    y = trial.suggest_float("y", -15, 30)

    v = x ** 2 + y ** 2

    # Prune if the objective values exceed the threshold.
    if v > 100:
        raise optuna.TrialPruned()

    return v


if __name__ == "__main__":
    study = optuna.create_study(
        storage="sqlite:///foo.db",
    )
    study.optimize(objective, n_trials=32, timeout=600)

    print("Number of finished trials: ", len(study.trials))
    print("Number of complete trials: ", len(study.get_trials(states=(optuna.trial.TrialState.COMPLETE,))))
```

## Context

Versions:
* optuna: 2.8.0.dev
* optuna-dashboard: 0.3.1

**Comment by c-bata** [comment]
Thank you for reporting! I fixed it at #70.

**Comment by toshihikoyanase** [comment]
Thank you for your swift action!
